src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,12 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args){
```

Comment:
Please ensure that there is a space between the closing parenthesis of the method signature and the opening curly brace `{` to adhere to Java code conventions.